addons/escoria-core/game/core-scripts/esc_room.gd

```diff
@@ -69,14 +69,12 @@ func _ready():
 		)
 		game.get_node("camera").set_target(player)
 	
-	if has_node("player_start"):
-		escoria.object_manager.register_object(
-			ESCObject.new(
-				$player_start.name,
-				$player_start
-			),
-			true
-		)
+	for n in get_children():
```

Comment:
Oooh. Nice! This is great to reduce ESC command use. It doesn't work for scenes with different exits, but it still makes things easier. 👍🏽

Reply:
Thanks :)